server/embedded/build.gradle

```diff
@@ -59,6 +59,48 @@ dependencies {
         }
     }
 
+    // TODO these shouldn't be merged forward
```

Comment:
Why should this not be merged forward?

Reply:
The force statements aren't needed with the spring-boot version used in develop, AFICT. Also, going forward as we move towards Embbedded do want to keep the standalone Spring version?